webapp/src/components/EventStatus.js

```diff
@@ -285,6 +285,20 @@ class EventStatus extends Component {
     }
   };
 
+  applicationOpeningStatus = (event) => {
+    return {
+      title: this.props.t("Applications Opening Soon"),
```

Comment:
By the way - whenever you add some new text to the UI, you also need to add the text to the [English](https://github.com/deep-learning-indaba/Baobab/blob/develop/webapp/public/locales/en/translation.json) and [French](https://github.com/deep-learning-indaba/Baobab/blob/develop/webapp/public/locales/fr/translation.json) translation files - that's what the `this.props.t` does - it looks up the phrase in the translation file for the user's selected language.

Reply:
Got it